django/contrib/contenttypes/models.py

```diff
@@ -137,6 +137,15 @@ def clear_cache(self):
 
     def _add_to_cache(self, using, ct):
         """Insert a ContentType into the cache."""
+        if not isinstance(ct, ContentType):
```

Comment:
This code reads a bit funny - is the ContentType to be inserted an instance of ContentType? It might be we don't have anything better for this, but if we have something in the ct._meta that would tell Django directly that this is a migration time model then this line would be easier to understand.